### DISCUSSION TITLE
[@types/react-native-metamap-sdk] Requesting TS package

### DISCUSSION THREAD

**Discussion by bombillazo** [DefinitelyTyped/DefinitelyTyped#59963] [Request a new @types package]
It would be great to get a TypeScript package for this library:

[react-native-metamap-sdk](https://github.com/GetMetaMap/mati-reactnative-plugin)

**Comment by jbovone** [comment]
It seems to be in typescript already.

**Reply by bombillazo** [reply]
Hey, I get this error when using the package though:

<img width="1011" alt="image" src="https://user-images.githubusercontent.com/44002321/173161991-85c0a75a-b00f-4560-b547-bd884c0d91b5.png">

**Comment by jbovone** [comment]
That's odd... It seems like trolling to me. 

Like... they ONLY have the index in js.
